README.md

```diff
@@ -8,26 +8,26 @@ The courtroom suite includes an abstract generic [SwearGame contract](#sweargame
 
 ### SwearGame contract
 
-This is the main general swear contract which conduct the trial.
+This is the main general swear contract which conducts the trial.
```

Comment:
You just go straight into terminology that has not been explained. What is "Swear contract"? What trial is being conducted? Sw3 has not been explained yet, so you can not use references to it.
When i said you should have a section that explains the terms and concepts being proposed this is what i meant.